installer/create-cluster-aks.sh

```diff
@@ -45,30 +45,30 @@ function parse_args {
         CLUSTER_NAME=$1
         [ -z "$CLUSTER_NAME" ] && error '"--cluster-name" requires a non-empty option argument.\n'
       ;;
-      --resource-group)
+      -r|--resource-group)
         shift
         RESOURCE_GROUP_NAME=$1
         [ -z "$RESOURCE_GROUP_NAME" ] && error '"--resource-group" requires a non-empty option argument.\n'
       ;;
       --location)
         shift
         LOCATION=$1
-        [ -z "$LOCATION" ] && error LOCATION=eastus
+        [ -z "$LOCATION" ] && error '"--location" requires a non-empty option argument.\n'
```

Comment:
I don't completely understand the phrase "a non-empty option argument."
Do you mean "a non-empty argument." ? 
(same x3 below)

Reply:
I think I would prefer ```--location" requires an argument.\n```